lib/angular/form/formController.js

```diff
@@ -5,32 +5,49 @@
 /**
  * The controller for the Form directive.
  *
- * This controller manages getting the form from the $fh.forms API using the AppformsClientService
+ * This controller manages getting the form from the $fh.forms API using the appformClient Service
  *
  * TODO: Implement Submission and Back Commands.
  *
  * @param $scope
- * @param AppformsClientService
+ * @param appformClient
+ * @param mediator
  * @constructor
  */
-function FormController($scope, AppformsClientService) {
+function FormController($scope, appformClient, mediator) {
   var formCtrl = this;
 
-  //TODO: mediator should be a service.
   formCtrl.submit = function() {
-    console.log("Submitting form", formCtrl.submission);
+    console.log("Submitting form locally", formCtrl.submission);
```

Comment:
This block of code looks like mostly debugging statements—should this be removed or is it valuable as a stub for  code that's coming?